pyscriptjs/src/utils.ts

```diff
@@ -84,11 +84,4 @@ function handleFetchError(e: Error, singleFile: string) {
     showError(errorContent);
 }
 
-/**
- * determines if the process is running inside the testing suite i.e. jest
- */
-function inJest(): boolean {
-    return process.env.JEST_WORKER_ID !== undefined;
-}
-
```

Comment:
Hmm, we can re-introduce later, what say? Otherwise it will be sitting here not being used at all.

Reply:
yes, but when it will be needed again, the poor soul will have to re-google and re-do the work which has already been done. Personally, I'd keep it, but I'll let you decide.

Reply:
I think I will re-introduce later if it's okay otherwise we can have linter issues again i.e. this function is not used anywhere etc.